Yii2Debug.php

```diff
@@ -96,8 +96,10 @@ public function init()
 
 		$panels = array();
 		foreach (CMap::mergeArray($this->corePanels(), $this->panels) as $id => $config) {
-			if (!isset($config['highlightCode'])) $config['highlightCode'] = $this->highlightCode;
-			$panels[$id] = Yii::createComponent($config, $this, $id);
+			if ($config) {
```

Comment:
I think, `enabled` param would be better.

```php
if (isset($config['enabled']) && !$config['enabled']) {
    continue;
}
```

So you will set `'enabled' => false` to any core panel, and this will exclude it.